tenant/config.go

```diff
@@ -0,0 +1,12 @@
+package tenant
```

Comment:
Why is this package not under `pkg` like the rest of the code?

Reply:
will move to under `pkg/`

Reply:
resolved, moved the new packages under the `pkg/` folder